src/main/resources/db/migration/V1__Create_Identification_Session_Table.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE identification_session
+(
+    id SERIAL PRIMARY KEY,
+    useid_session_id UUID NOT NULL UNIQUE,
+    refresh_address VARCHAR NOT NULL,
+    request_data_groups VARCHAR(5)[] NOT NULL,
+    eid_session_id  UUID UNIQUE
+);
```

Comment:
I think we are missing the index to search for useid_session_id and eid_session_id

Reply:
I thought about adding those indices but decided against it. The reason was that an index makes inserting and updating an entity slower whilst it speeds up searching for one. We do insert and update as often as we search for entities which is why I though we might not need indices. Furthermore, I read an article which said that adding an index on a column which contains a lot of null values (eidSessionId) is not recommended. Unfortunately it was not explained.

Reply:
Yes, updating of the values/columns that are in the index, which I don't think we do as much. Given this is a session storage and the amount of sessions is likely to be small < 1000 it probably doesn't matter too much at this point and we can always add indices later on. Can you please add a comment to the SQL file about the considerations, because I'm probably not the only one that will jump on this :-)

Reply:
Also, speed wise, everything will be in memory anyways, considering how small the data set is and how we sized the database for production. So I don't think there will be a noticeable performance difference in inserts and updates

Reply:
Ok but if it does not hurt, we could also just add indices instead of a comment.

Reply:
Seems like [PostreSQL automatically creates indices for columns with a unique constraints](https://www.postgresql.org/docs/8.1/sql-createtable.html#SQL-CREATETABLE-NOTES).
> PostgreSQL automatically creates an index for each unique constraint and primary key constraint to enforce uniqueness. Thus, it is not necessary to create an index explicitly for primary key columns. (See [CREATE INDEX](https://www.postgresql.org/docs/8.1/sql-createindex.html) for more information.)